section4/exercises/classMethods.js

```diff
@@ -28,13 +28,37 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
-  // ADD CODE
-};
+  changeProtein(newProtein) {
+    if (typeof newProtein === 'string') {
+      this.protein = newProtein
+    }
+  }
+
+  addTopping(newTopping) {
+    if (typeof newTopping === 'string') {
+      this.toppings = this.toppings.concat(newTopping);
```

Comment:
Nice job with dynamic methods that use a parameter to get input. 
The concat method is not the one typically used to add an item to an array, but rather to merge 2 arrays. Research prototype methods for arrays in javascript, and play around with some in repl as practice for Mod1.

Reply:
Yeah, I'd love some additional insight here. I originally did `array.push`, but then it would just return the length of the array. Is `array.push` the code you were thinking of or should I keep diggin?